tunedev/Banka-react
# #167122079 Add functionality to the create account page

#### What does this PR do?
Adds functionality to the create account page

#### Description of Task to be completed?
> - add create account action
> - add account state to the store
> - add test for functionality

#### How should this be manually tested?
> - Clone this branch
> - Then run `npm install` and `npm test'
> - Then run `npm run dev' 
> - Click on the Sign-up button on the Landing page
> - Fill the form with unique details and hit submit to sign up
> - Then navigate to the new account page and fill the form
> - You should get a message that the account has been created

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#167122079](https://www.pivotaltracker.com/story/show/167122079)

#### Screenshots (if appropriate)
NA
#### Questions:
NA
